kyuubi-common/src/test/scala/org/apache/kyuubi/util/SparkVersionUtil.scala

```diff
@@ -17,13 +17,22 @@
 
 package org.apache.kyuubi.util
 
-import org.apache.kyuubi.SPARK_COMPILE_VERSION
```

Comment:
it should be a mistake, it does not make sense to check compile version.

Reply:
I think it could represent the Spark compiled version, but we need a runtime version here.